src/triage/component/results_schema/README.md

```diff
@@ -7,31 +7,35 @@ Store results of modeling runs in a relational database
 
 `pip install git+https://github.com/dssg/triage.git`
 
-2. Create a YAML file with your database credentials (see example_db_config.yaml), or an environment variable 'DBURL' with a connection string. The database must be created already.
+2. You can do the initial schema and table creation a couple of different ways, with a couple of different options for passing database credentials.
 
-3. Call 'upgrade_db' function from Python console or script
+	- *Triage cli script*. Assuming you have triage installed from pip: `triage upgrade_db -d dbcredentials.yaml`, or just `triage upgrade_db` if you have a database credentials file at `database.yaml` (the default)
```

Comment:
I think with how `argparse` does subcommands it'd be:

    triage [-d dbcredentials.yaml] updatedb

(Also, BTW, I know @tweddielin is working on how best to parse that optionally-specified path and hand it off within all the subcommands that want it.)